### ISSUE TITLE
Multi-Thumb Slider: Has no single pointer alternative to dragging

### ISSUE THREAD

**Issue by adampage** [w3c/aria-practices#3235]
While doing a PR review on #3214, I noticed that the current [Multi-Thumb Slider](https://www.w3.org/WAI/ARIA/apg/patterns/slider-multithumb/examples/slider-multithumb/) pattern does not satisfy WCAG 2.2’s [2.5.7 Dragging Movements](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html) success criterion. A user can only set values by keyboard or with a dragging gesture; there is no single pointer method available.

In contrast, the [Vertical Temperature Slider](https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-temperature/) and [Media Seek Slider](https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-seek/) _do_ offer a single pointer alternative to dragging, although there is an open issue challenging its current behavior (#2446).

**Comment by adampage** [comment]
It certainly did, @mcking65, sorry about that. 🤦🏻  I noticed it when reviewing [a different PR](https://deploy-preview-381--aria-practices.netlify.app/aria/apg/patterns/slider-multithumb/examples/slider-multithumb/) where that fix evidently hadn’t been merged in; I should have checked the production deployment. Please forgive my false alarm!
